src/parcsr_ls/par_relax.c

```diff
@@ -785,6 +786,7 @@ hypre_BoomerAMGRelaxHybridGaussSeidel_core( hypre_ParCSRMatrix *A,
 #if defined(HYPRE_USING_PERSISTENT_COMM)
       hypre_ParCSRPersistentCommHandleStart(persistent_comm_handle, HYPRE_MEMORY_HOST, v_buf_data);
 #else
+      hypre_ParCSRCommHandle *comm_handle;
```

Comment:
Why do we need to define another `comm_handle` here?

Reply:
This is not needed, thanks for catching!

Reply:
Fixed with [e082daf](https://github.com/hypre-space/hypre/pull/739/commits/e082dafda6a2ef544eda2fca27d62141d433872b)